Arrangement_on_surface_2/include/CGAL/Curved_kernel_via_analysis_2/gfx/Curve_renderer_internals.h

```diff
@@ -90,8 +90,11 @@ struct Pixel_2_
 
     Integer sub_x, sub_y; // subpixel coordinates relative to pixel's boundary
                           // (always 0 for pixels)
+    Pixel_2_() = default;
 
-    Pixel_2_& operator =(const Pixel_2_& pix) = default;
+    Pixel_2_(const Pixel_2_&) = default;
+
+    Pixel_2_& operator =(const Pixel_2_&) = default;
```

Comment:
An easier way would be to remove all the special member functions, and let the compiler generate them. Without any special member function, that `struct` becomes a [POD type](https://en.cppreference.com/w/cpp/named_req/PODType).